examples/heartbeat_caravel_wrapper/tools/addvias/addvias.py

```diff
@@ -7,12 +7,14 @@ def make_docs():
 
 def setup(chip):
     tool = 'addvias'
-    design = chip.get('design')
+    design = chip.get('design')x
```

Comment:
remove `x` (although this test clearly isn't in the CI)

Reply:
Clearly, will fix:-)

Reply:
It's a daily test so would have broke those :)